packages/stream-metadata/src/routes/spaceRefresh.ts

```diff
@@ -41,9 +45,17 @@ export async function spaceRefreshOnResponse(
 
 	logger.info({ spaceAddress }, 'Refreshing space')
 
+	const { target } = querySchema.parse(request.query)
+	const paths =
+		target === 'metadata'
+			? [`/space/${spaceAddress}`, `/space/${spaceAddress}/token/*`]
+			: target === 'image'
+			? [`/space/${spaceAddress}/image`]
+			: [`/space/${spaceAddress}/*`]
```

Comment:
for "all" will `/space/${spaceAddress}/*` match `/space/${spaceAddress}`? I am superstitious about trailing slashes.

Reply:
https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/invalidation-specifying-objects.html#:~:text=If%20you%20use%20the%20AWS,a%20path%20is%204%2C000%20characters.

to include the `/space/spaceAddress` we could use `/space/spaceAdress*`

Reply:
`/*` only match things inside the slash